InternshipBE/ExaLearn.Dal/Database/DbInitializer.cs

```diff
@@ -1,20 +1,148 @@
 ﻿using ExaLearn.Dal.Entities;
+using ExaLearn.Dal.Interfaces;
 using Microsoft.AspNetCore.Identity;
 using Portal.Core.Constants;
+using Shared.Enums;
+using System.Collections.Generic;
 using System.Linq;
 using System.Threading.Tasks;
 
 namespace ExaLearn.Dal
 {
     public static class DbInitializer
```

Comment:
![image](https://user-images.githubusercontent.com/71830071/125842743-13e56574-b357-4b4e-86f2-68e0a5ae10d5.png)
move

Reply:
when you execute update-database db should be filled